Glyssen/Dialogs/ScriptureRangeSelectionDlg.cs

```diff
@@ -65,6 +67,41 @@ private void Initialize()
 				m_checkBoxOldTestament.Checked = false;
 			if (!m_includeInScript.Any(p => p.Value && BCVRef.BookToNumber(p.Key) > 39))
 				m_checkBoxNewTestament.Checked = false;
+
+			SetupDropdownHeaderCells();
+		}
+
+		private void SetupDropdownHeaderCells()
+		{
+			// set up dropdown column headers
+			var columnOptions = new SortedList<int, string>
+			{
+				{0, LocalizationManager.GetString("DialogBoxes.ScriptureRangeSelectionDlg.SelectAll", "Select All")},
+				{1, LocalizationManager.GetString("DialogBoxes.ScriptureRangeSelectionDlg.SelectAll", "Clear All")}
+			};
+
+			m_ntBooksGrid.Columns[2].HeaderCell = CreateDropdownHeaderCell(columnOptions);
+			m_ntBooksGrid.Columns[3].HeaderCell = CreateDropdownHeaderCell(columnOptions);
+			m_otBooksGrid.Columns[2].HeaderCell = CreateDropdownHeaderCell(columnOptions);
+			m_otBooksGrid.Columns[3].HeaderCell = CreateDropdownHeaderCell(columnOptions);
+		}
+
+		private DataGridViewDropdownColumnHeaderCell CreateDropdownHeaderCell(SortedList<int, string> columnOptions)
+		{
+			var cell = new DataGridViewDropdownColumnHeaderCell(m_ntBooksGrid.Columns[2].HeaderCell, columnOptions);
```

Comment:
Don't we need to pass in the HeaderCell here rather than it being m_ntBooksGrid.Columns[2] for all of them?